keps/sig-storage/20190204-kubernetes-csi-release-process.md

```diff
@@ -60,35 +59,39 @@ This document explains how these components are released.
 
 So far, the process for tagging and building components has been
 fairly manual, with some assistance by Travis CI. There has been no
-automatic end-to-end testing.
+automatic testing on a Kubernetes cluster, neither stand-alone
+deployment of individual components nor full end-to-end (E2E) testing
+of several Kubernetes-CSI components.
 
 With CSI support reaching GA in Kubernetes 1.13, it is time to define
 and follow a less work-intensive and more predictable approach.
 
 ### Goals
 
-- define what a "kubernetes-csi release" is
-- unit testing as mandatory pre-submit check for all components 
-- E2E testing as mandatory pre-submit check for components that get
+- define the release process for each component
+- unit testing as mandatory pre-submit check for all components
+- testing on Kubernetes as mandatory pre-submit check for components that get
   deployed in a cluster
-- define a release process for each component and a "kubernetes-csi release"
 
 ### Non-Goals
 
-- change responsibilities among the kubernetes-csi maintainers - as before, each
+- a combined "Kubernetes-CSI release": each component gets released separately.
+  It is the responsibility of a CSI driver maintainer pick and test sidecar releases
```

Comment:
We could mention that the hostpath deployment example can be used as a recommendation/starting point, but it's not going to be able to test all the possible combinations of features that a specific CSI driver may need.

Reply:
Sure.